esmvaltool/preprocessor/_regrid.py

```diff
@@ -508,8 +506,8 @@ def get_reference_levels(filename,
     cubes = fix_metadata(cubes, short_name, project, dataset)
     cube = cubes[0]
     try:
-        coord = cube.coord(coordinate)
+        coord = cube.coord(coord)
```

Comment:
```suggestion
        coord = cube.coord(axis='Z')
```
and remove the function argument

Reply:
yes, you are falling for the same I did yesterday and reverted the changes, the tests are done with a non-descriptive z axis, and if you change the test a whole lot of other crap starts falling; just leave it this way and the user can give it any (working) value

Reply:
Just fix the tests instead, cmorized data should have the correct z axis, right?

Reply:
I think so but am not 100% sure - @jvegasbsc ?